modules/programs/jq.nix

```diff
@@ -0,0 +1,68 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.jq;
+
+  colorType = mkOption {
+    type = types.str;
+    description = "ANSI color definition";
+    example = "1;31";
```

Comment:
I think you could change this to
```nix
  colorType = mkOption {
    type = types.str;
    visible = false;
  };
```
to suppress the individual options from showing up in the documentation. The repetitiveness is not very useful, I think the example suffices to show the available color options.